config/storybook/decorators.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Grid, Paper, Table, TableRow, TableCell, TableHead, TableBody } from "@material-ui/core";
+
+
+
+/** A custom "CombinationRender" for react-storybook-addon-props-combinations addon */
+export const CombinationRenderer = ({ Component, props }) => {
+  const propNames = Object
+    .keys(props)
+    .filter((key) => key !== "children" && typeof props[key] !== "function");
+
+  return (
```

Comment:
There's no way to tell whether this is going to work since we have no components.  I'm willing to leave it for now and wait until we have a component to check for any bugs.  The alternative would be to test the `FakeComponent`, but that won't buy us much since it's just a `span`.

Reply:
I can pull up the empty UI, I think it's fine.